sdk/storage/.editorconfig

```diff
@@ -0,0 +1,136 @@
+# Rules in this file were initially inferred by Visual Studio IntelliCode from the C:\azure-sdk-for-net\sdk\storage codebase based on best match to current usage at 12/10/2021
```

Comment:
I dont even know how I added this. sigh let me fix this.

Reply:
much appreciated.